html-forms/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <p>Email
+      <br/>
+      <input type="text" name="email" />
```

Comment:
All inputs that have text content should be using a label element either by using a "for" attribute to connect the two, or by wrapping the input inside a label.